server/src/domain/map/map-styles.dto.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsEnum, IsOptional, IsString } from 'class-validator';
+
+enum MapTheme {
+  LIGHT = 'light',
+  DARK = 'dark',
+}
+
+export class MapStylesDto {
+  @IsString()
+  name!: string;
+
+  @IsString()
+  url!: string;
+
+  @IsEnum(MapTheme)
+  @IsOptional()
+  @ApiProperty({ type: String, enum: MapTheme, enumName: 'MapTheme', required: false })
+  theme!: 'light' | 'dark' | undefined;
```

Comment:
This is not required? Then it should be `theme?: MapTheme`

Reply:
I thought since this is also a request dto, it would need to be `theme!: ...`?